scripts/train.py

```
@@ -0,0 +1,151 @@
+        try:
+            processed_data = engineer_features(raw_data)
+            available_features = [f for f in self.features if f in processed_data.columns]
+            
+            if len(available_features) != len(self.features):
+                missing = set(self.features) - set(processed_data.columns)
+                print(f"Warning: Missing features {missing} - using available features")
+                
+            scaled_data = self.scaler.transform(processed_data[available_features])
+            return float(self.model.predict_proba(scaled_data)[0, 1])
+            
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)